Adyen/CheckoutConfiguration/CheckoutConfiguration.swift

```diff
@@ -47,18 +47,13 @@ public struct CheckoutConfiguration {
         self.amount = amount
         self.analyticsConfiguration = analyticsConfiguration
         
-        var configDictionary: [CheckoutComponentType: ConfigurationWrapper] = [:]
+        var configDictionary: [CheckoutComponentType: CheckoutComponentConfiguration] = [:]
```

Comment:
It would help to have a more descriptive name. Such as `componentConfigurations`. Just an idea

Reply:
It's a local var assigned to actual `configurations` property at the end. IMO it's fine